lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -29,6 +29,7 @@ use Zonemaster::Backend::Validator;
 
 my $zm_validator = Zonemaster::Backend::Validator->new;
 my %json_schemas;
+our %extra_validators;
```

Comment:
Any reason for "our" instead of "my"?

Reply:
I need to access this from the tests, I am not sure if an accessor function would be better or if that is okay to do?

Reply:
It is only used in RPCAPI.pm, isn't it? And that is the same lexical scope.

Reply:
It is used RPCAPI.pm and test_validate_syntax.t https://github.com/zonemaster/zonemaster-backend/blob/dd09207b2d9dccd5a0a4ded68890e56edc6f4937/t/test_validate_syntax.t#L33

Reply:
Ok.